waku/v0/init.go

```diff
@@ -0,0 +1,5 @@
+package v0
+
+func Init() {
+	initRLPKeyFields()
+}
```

Comment:
We don't need to export this function, `init()` will fire automatically any time that Go accesses the package.

Reply:
ah, nice, I didn't know that, thanks!